docs/resources/tutorial-resources.md

```diff
@@ -0,0 +1,79 @@
+
+We recommend following the [Software Carpentry Curriculum Development](https://carpentries.github.io/curriculum-development/) model for building hackweek tutorials. The core components of this "Backward Design" approach include:
+
+```{note}
+1. Start by identifying key learning outcomes
+2. Identify the practical skills that would guide participants to these outcomes
+3. Build challenges that give learners practice in integrating new skills
+```
+
+### Lessons learned from previous hackweeks
+
+* *Keep it simple and brief!* Hackweeks have a lot going on, and between tutorials, networking events, and projects there is a lot for participants to digest.
+
+* Make your tutorials *interactive* through live coding challenges, working through a Juypter Notebook together, or interactive discussions. Leave some space when you are not talking for people to work through examples and ask questions.
+
+* Try for content that is *broad in scope* and gives an initial picture of what might be possible when applying a particular data science tool. Save detailed explanations on a narrow topic for optional breakout sessions.
+
+* *Start with simple explanations* that attend to people who are seeing this content for the first time, AND *keep advanced participants engaged by inviting them to assist others*, or to explore more advanced concepts through individual study.
+
+
+### Formatting
+
+An effective format for tutorial content is in the form of a Jupyter Notebook that provides participants with narrative text and interactive code examples. Notebooks can contain a wide variety of embedded media (images, videos, equations, etc.) and can be easily rendered on hackweek event websites.
+
+We recommend that all notebooks make use of sections with the following minimal structure:
+```
+# Title
+* Learning objectives *
+## Software
+## Data
+## My Section
+### My Subsection
+## Summary
+## References
+```
+
+For convenience you can start with a copy of our [template notebook](https://uwhackweek.github.io/jupyterbook-template/tutorials/example/tutorial-notebook.html), which you can download [here](https://github.com/uwhackweek/jupyterbook-template/raw/main/book/tutorials/example/tutorial-notebook.ipynb).
+
+
+#### Examples
+
+Spend a bit of time looking at these examples from past events:
+
+| Description | Links |
+| - | - |
+| SnowEx 2021 Thermal IR | (rendered)[https://snowex-hackweek.github.io/website/tutorials/thermal-ir/thermal-ir-tutorial.html], (raw)[https://github.com/snowex-hackweek/website/raw/main/book/tutorials/thermal-ir/thermal-ir-tutorial.ipynb] |
+
+
+## Delivering a tutorial
+
+Prepare your tutorials well in advance and practice it in front of a small audience for feedback. This helps ensure you have enough time to cover your full tutorial, can be an opportunity for feedback, and will give you confidence when presenting during the hackweek.
+
+### Examples
```

Comment:
Should these examples be combined with the above and live as a dedicated section? Each subsection can say something like “See samples from past events” and link to that.